ores/ores.py

```diff
@@ -6,6 +6,11 @@
 * celery_worker -- Starts a "ScoreProcessor" celery worker
 * precached -- Starts a daemon that requests scores for revisions as they happen
 
+You can also launch a set of production like applications
+
+* applications.wsgi -- A wsgi server
+* applications.celery -- A celery worker
+
```

Comment:
A decision made long ago. See all of the rest of the utilities in this package.  If you'd like to discuss this pattern, it seems that it would be easier to start a whole separate thread.